spec/features/admin/admin_merchant_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe "Admin Merchant Page", type: :feature do
+  describe "When I visit the merchant index (/admin/merchants)" do
+    # US 24
+    it "I a list of all the merchants" do
+      merchant_1 = Merchant.create!(name: "Merchant 1")
+      visit admin_merchants_path
+
```

Comment:
you might want to add more merchants here so you can test for more than one

Reply:
I agree with @z-fitch. If we want to test that it lists ALL the merchants, I think we should have more than one listed.

Reply:
done